charts/federation-v2/templates/crds.yaml

```diff
@@ -0,0 +1,1200 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    "helm.sh/hook": crd-install
```

Comment:
(Suggestion) As discussed in wg meeting, it would be better additionally to add a script to generate CRD definitions from the code and add customised stuff like adding annotations.

Reply:
Yes, I will follow up this task.